config/routes.rb

```diff
@@ -8,19 +8,22 @@
 
     # Patient routes
     # /patients/:id
-    # /patients/:id/calls 
+    # /patients/:id/calls
     # /patients/:id/notes
     # /patients/:id/external_pledges
-    resources :patients, 
+    resources :patients,
               only: [ :create, :edit, :update ] do
-      resources :calls, 
+      resources :calls,
                 only: [ :create, :destroy ]
-      resources :notes, 
+      resources :notes,
                 only: [ :create, :update ]
-      resources :external_pledges, 
+      resources :external_pledges,
                 only: [ :create, :update, :destroy ]
     end
 
+    get 'naf_status', to: 'clinics#naf_status', as: 'naf_status'
```

Comment:
Do we need this endpoint? Javascript is admittedly not my strong suit but it doesn't look like it's getting called anymore. Thoughts?